plugins/generator-1.21.4/neoforge-1.21.4/templates/json/model_item_shield.json.ftl

```diff
@@ -1,14 +1,39 @@
+<#if data.hasGUITexture?? && data.hasGUITexture()>
 {
   "model": {
-    "type": "minecraft:condition",
-    "on_false": {
-      "type": "minecraft:model",
-      "model": "${modid}:item/${registryname}"
-    },
-    "on_true": {
-      "type": "minecraft:model",
-      "model": "${modid}:item/${registryname}_blocking"
-    },
-    "property": "minecraft:using_item"
+    "type": "minecraft:select",
+    "property": "minecraft:display_context",
+    "cases": [
+      {
+        "when": ["gui", "fixed"],

```

Comment:
This texture could also be applied when the item is dropped (`"ground"`), as is the case with trident and spyglass

Reply:
[b70f0df](https://github.com/MCreator/MCreator/pull/5459/commits/b70f0df2a182e318627bae157c9df31f5f92ffa0)